### PR TITLE
Allow using same business key for different tenants

### PR DESCRIPTION
**Pull request by EugeneKostrikov**

### CODE REVIEW

File: test/fortune/multitenant-integration.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        ]}))
+        .expect(201)
+        .end(function(err){
+          should.not.exist(err);
+          request(baseUrl).post('/people')
+            .set('content-type', 'application/json')
+            .send(JSON.stringify({people: [
+              {email: 'dilbert@mailbert.com', _tenantId: 'testing'}
+            ]}))
+            .expect(500)
```
Comment (dmskr) [inline test/fortune/multitenant-integration.js]: Something like status 400-409 (bad request or conflict) is more appropriate here i think.

Comment (EugeneKostrikov) [inline test/fortune/multitenant-integration.js]: It's not implemented yet - there's outstanding ticket addressing error handling. Atm it's setting status 500 for any type of error that happens in adapter and it seems to be most suitable one as it's more likely to be some database connection failure.

Comment (dmskr) [inline test/fortune/multitenant-integration.js]: Mm. The failure happens (should happen) if client tries to create record, duplicating primary key. So it doesn't look like database connection issue and more related to the request itself.
Ok

Comment (EugeneKostrikov) [inline test/fortune/multitenant-integration.js]: Yep, exactly. And we have no way to figure out if it was duplicate primary key or connection failure atm. https://github.com/flyvictor/fortune/issues/150

Comment (dmskr) [inline test/fortune/multitenant-integration.js]: Mongo rises mongo-specific errors, which can be caught and recognized correctly in the adapter. Anyway. Thanks for the link.

**Pull request merged**